arm/cdn/customdomains.go

```diff
@@ -118,14 +122,18 @@ func (client CustomDomainsClient) CreateResponder(resp *http.Response) (result a
 	return
 }
 
-// DeleteIfExists sends the delete if exists request.
+// DeleteIfExists sends the delete if exists request.This method handles
```

Comment:
@garimakhulbe there should be a space after period `.` 

also the following sentence didn't seem grammatically correct:

    User can make polling optional for long-running operations

I don't understand what it means. I believe `cancel` channel is there for cancellation (not for avoiding LRO entirely) right?